docs/3-administrator-guides/06-standard-modules/03-standard-widgets/README.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Widgety Standardowe
+---
+
+Widgety w YetiForce CRM to interaktywne elementy, które można umieszczać na pulpicie nawigacyjnym systemu. Umożliwiają one szybki dostęp do najważniejszych informacji i funkcji, a także personalizację interfejsu użytkownika zgodnie z indywidualnymi potrzebami.
+
+Głównym celem stosowania widgetów w YetiForce CRM jest:
+
+* Zwiększenie produktywności: Widgety pozwalają użytkownikom na szybki dostęp do kluczowych informacji i funkcji bez konieczności przechodzenia przez rozbudowane menu.
+* Personalizacja interfejsu: Użytkownicy mogą wybierać i układać widgety na pulpicie nawigacyjnym tak, aby odpowiadało to ich preferencjom i sposobowi pracy.
+* Dostęp do danych w czasie rzeczywistym: Wiele widgetów wyświetla aktualne dane, takie jak liczba leadów, stan zadań itp.
+* Ułatwienie nawigacji: Widgety mogą służyć jako skróty do często używanych modułów lub funkcji systemu.
+
+Część widgetów jest predefiniowana i od razu gotowa do użycia. Jeżeli jednak potrzebujemy wyświetlić niestandardowe dane, możemy stworzyć widget według własnych preferencji.
+
+
+## [Widgety Predefiniowane](/administrator-guides/standard-modules/standard-widgets/predefined-widgets)
+
+Widgety predefiniowane w YetiForce CRM to gotowe do użytku elementy, które można umieścić na pulpicie nawigacyjnym systemu. Zostały one zaprojektowane przez zespół YetiForce i oferują szeroki zakres funkcji, które ułatwiają użytkownikom pracę z systemem.
+
+
+## [Widgety z filtrem](/administrator-guides/standard-modules/standard-widgets/widgets-from-filters)
```

Comment:
To można zamienić na
 ```
import DocCardList from '@theme/DocCardList';
```
Wtedy automatycznie wygeneruje listę z opisem  z pozostałych plików md